A6-client-retries.md

```diff
@@ -339,11 +343,11 @@ This is an example of a retry policy and its associated configuration. It implem
 
 #### Hedging Policy
 
-The following example of a hedging policy configuration will issue up to three hedged requests for each RPC, spaced out at 500ms intervals, until either: one of the requests receives a valid response, all fail, or the overall call deadline is reached. Analogously to `retryableStatusCodes` for the retry policy, `nonFatalStatusCodes` determines how hedging behaves when a non-OK response is received.
+The following example of a hedging policy configuration will issue an original RPC, then up to three hedged requests for each RPC, spaced out at 500ms intervals, until either: one of the requests receives a valid response, all fail, or the overall call deadline is reached. Analogously to `retryableStatusCodes` for the retry policy, `nonFatalStatusCodes` determines how hedging behaves when a non-OK response is received.
```

Comment:
> Analogously to `retryableStatusCodes` for the retry policy, `nonFatalStatusCodes` determines how hedging behaves when a non-OK response is received.

Off-topic: maybe we should discuss renaming `retryableStatusCodes` to `nonFatalStatusCodes` for even more consistency between hedging/retries?

Reply:
Let's save this for a different thread if we want to make the change